src/commands/EjectCommand.js

```diff
@@ -0,0 +1,47 @@
+import Command from "../Command";
+import FileSystemUtilities from "../FileSystemUtilities";
+import async from "async";
+import path from "path";
+import userHome from "user-home";
+
+export default class EjectCommand extends Command {
+  initialize(callback) {
+    this.to = this.flags.to || userHome;
+    this.dry = this.flags.dry;
+
+    this.pkg = this.input[0];
+
+    if (this.dry) {
+      this.logger.info("The following directories would be moved");
```

Comment:
In the case where you're exporting more than one module, you'd log "The following directories would be moved" before each of them instead of at the end.  I could instead gather all the strings in an accumulator that is passed to each callback, and log it after all the asynchronous calls to visit the packages has completed, which would also clean up the unit tests, so I'll do that ;)